### DISCUSSION TITLE
Ensembl GFF transcript "click"

### DISCUSSION THREAD

**Discussion by grexor** [GMOD/jbrowse-components#4363] [General]
Hello everyone, great work! Simple question:

I am adding the Ensembl GFF file and would like to be able to click on individual transcripts to display the information, however now the whole "gene" with all the transcripts is one "click". Possible to un-group this?

I am doing this:

```
wget https://ftp.ensembl.org/pub/release-111/gff3/homo_sapiens/Homo_sapiens.GRCh38.111.gff3.gz
gunzip Homo_sapiens.GRCh38.111.gff3.gz
jbrowse sort-gff Homo_sapiens.GRCh38.111.gff3 | bgzip > Homo_sapiens.GRCh38.111.gff3.gz
tabix --force Homo_sapiens.GRCh38.111.gff3.gz
jbrowse add-track Homo_sapiens.GRCh38.111.gff3.gz --load copy --out ... --force --name "Ensembl 111" --category "Genes"
```

Thanks for any hints and sorry if I overlooked how to do this in the docs,

Best,
Gregor

**Comment by cmdcolin** [comment]
currently there is no way to do this without manually adjusting the gff3 file to remove "gene" lines

note that we do have some active feature requests for being able to select features from within the single gene glyph here, i think it would be a good idea

https://github.com/GMOD/jbrowse-components/issues/561

**Comment by grexor** [comment]
Thanks! I tried to remove the "gene" records, however now I get:

![image](https://github.com/GMOD/jbrowse-components/assets/1798838/ece22822-bf3d-402f-b878-d1f9ad8bb644)

Some other ideas of how to easily fix this? Thanks, Gregor

**Comment by cmdcolin** [comment]
i made a short script here that can strip the "Parent=" attribute from the child transcripts after the gene lines are removed https://github.com/cmdcolin/strip-gene-from-gff

**Comment by grexor** [comment]
Thanks very useful, will probably write also something that moves the Parent=gene: to the name (to keep the ENSG gene id).

`Parent=gene:ENSG00000143776;Name=CDC42BPA-209;` -> `Name=ENSG00000143776,CDC42BPA-209;` or something in this direction.

**Comment by grexor** [comment]
Now here: https://github.com/grexor/pybio

`pybio gff4jbrowse input.gff output.gff`

<img width="909" alt="image" src="https://github.com/GMOD/jbrowse-components/assets/1798838/7b39b476-e8ef-43bc-95e9-90503e896f2b">
